lerna.json

```diff
@@ -0,0 +1,7 @@
+{
+  "lerna": "2.0.0-rc.4",
```

Comment:
Stick with lerna 2.0.0-rc.4 so your on the same version as the other repos. Theirs a build failure in lerna 2.0.0 and the core maintainer is on vacation, should be resolved when he gets back then will update to v2.0.0